src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -361,7 +361,7 @@ private void addAndSearch(@NotNull final World world, @NotNull final BlockPos lo
                 for (int z = -1; z <= 1; z++)
                 {
                     final BlockPos temp = log.add(x, y, z);
-                    if (BlockPosUtil.getBlock(world, temp).isWood(null, temp) && !woodBlocks.contains(temp))
+                    if ((BlockPosUtil.getBlock(world, temp).isWood(null, temp) || world.getBlockState(temp).getMaterial() == Material.CLAY ) && !woodBlocks.contains(temp) && world.getBlockState(temp).getBlock() != Blocks.CLAY)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 226 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)